base/server/src/main/java/com/netscape/cmscore/apps/CMS.java

```diff
@@ -53,6 +54,8 @@ public final class CMS {
 
     private static CMSEngine engine;
 
+    private static SSLCertificateApprovalCallback approvalCallback;
```

Comment:
In the newer branches we've removed the dependency on global variables in `CMS` class. I'd suggest moving this variable into `CMSEngine` class.